src/Microsoft.ComponentDetection.Orchestrator/Experiments/Configs/UvLockDetectorExperiment.cs

```diff
@@ -0,0 +1,23 @@
+namespace Microsoft.ComponentDetection.Orchestrator.Experiments.Configs;
+
+using Microsoft.ComponentDetection.Contracts;
+using Microsoft.ComponentDetection.Detectors.Pip;
+using Microsoft.ComponentDetection.Detectors.Uv;
+
+/// <summary>
+/// Experiment to validate UvLockComponentDetector against PipComponentDetector.
+/// </summary>
+public class UvLockDetectorExperiment : IExperimentConfiguration
```

Comment:
Looks like this should have been `DefaultOffComponentDetector` to begin with https://github.com/microsoft/component-detection/blob/main/docs/creating-a-new-detector.md#detector-lifecycle.